policyengine_us/variables/gov/states/az/tax/income/subtractions/college_savings/college_savigns_plan_529_benefitiary.py

```diff
@@ -0,0 +1,8 @@
+from policyengine_us.model_api import *
+
+
+class college_savigns_plan_529_benefitiary(Variable):
```

Comment:
@MaxGhenis we have created this variable to represent a beneficiary of the subtraction, let us know if the format works

Reply:
the subtraction is limited to people who both contribute to and benefit from (withdraw) 529s in the same year?

if this is correct then please fix the typos in filename and seems like it should be bool rather than int

Reply:
the subtraction is limited to tax units which have contributed to the 529 plan and the maximum amount is dependent on the number of beneficiaries 

![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/29562ce0-338c-465e-8378-cd4df16066a3)

Reply:
ok rename `count_529_contribution_beneficiaries` and move to the same folder as `investment_in_529_plan` since it's not az-specific

btw that's a segment from the ABLE account subtraction, which looks to have an identical structure